aries-site/src/layouts/content/UpdateNotification.js

```diff
@@ -0,0 +1,52 @@
+import { Anchor, Text, Notification } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+import { ViewContext } from '../../pages/_app';
+import { useContext } from 'react';
+
+function InlineNotification({ dateText, message, updateDate }) {
+  return (
+    <Notification
+      width="large"
+      title={
+        dateText +
+        new Date(updateDate).toDateString().split(' ').slice(1).join(' ')
+      }
+      message={message}
+      margin={{ bottom: 'medium' }}
+      icon={<CircleInformation />}
+    />
+  );
+}
+
+export const UpdateNotification = ({ name }) => {
+  const { contentHistory } = useContext(ViewContext) || undefined;
+  const updateDate = contentHistory[name]?.date;
+
+  if (contentHistory && name in contentHistory) {
```

Comment:
If the desire here is to check that `name` key exists and has a value, this is another way to write that.

Just flagging, but doesn't necessarily need to change.

```suggestion
  if (contentHistory?.name) {
```